### ISSUE TITLE
Bug: TalkBack audio ducking breaks playback

### ISSUE THREAD

**Issue by alexstine** [Automattic/pocket-casts-android#829]
### Description

When TalkBack audio ducking is enabled, the audio gets paused in a way while the screen reader is speaking and only when it finishes does the audio start playing. The audio should simply lower, not pause with audio ducking.

### Step-by-step reproduction instructions

1. Enable TalkBack.
2. Enable audio ducking.
3. Open Pocket Casts.
4. Choose a podcast episode to play and select Play.
5. Notice that when TalkBack is speaking, you cannot hear the episode audio until TalkBack finishes speaking.
6. This is not an issue if you disable TalkBack audio ducking.
7. I would expect this bug to not exist. Episode audio should lower with audio ducking, not stop.

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

Google Pixel 7 with Pocket Casts 7.33.

**Comment by mchowning** [comment]
👋 @alexstine ! If you change the "play over nofications" setting in Pocket Casts (accessible within the settings under "Notifications"), you'll be able to get the behavior you want. 

I see the behavior you observe when that setting is "Never", but if I set that to either "Duck volume" or "Always", it does not pause for me.

**Comment by alexstine** [comment]
That fixed it, thanks.
